openpype/hosts/max/plugins/publish/extract_max_scene_raw.py

```diff
@@ -19,7 +19,8 @@ class ExtractMaxSceneRaw(publish.Extractor,
 
     order = pyblish.api.ExtractorOrder - 0.2
     label = "Extract Max Scene (Raw)"
-    hosts = ["max"]
+    hosts = ["max",
+             "maxScene"]
     families = ["camera"]
```

Comment:
```suggestion
    hosts = ["max"]
    families = ["camera", "maxScene"]
```
I suspect this would fix the bug mentioned by @LiborBatek

Reply:
my bad to set it wrong. will update in the next PR